homeassistant/components/trello/const.py

```diff
@@ -0,0 +1,15 @@
+"""Constants for the trello_int integration."""
+import logging
+from typing import Final
+
+DOMAIN: Final = "trello"
+
+LOGGER = logging.getLogger(__package__)
+
+CONF_API_TOKEN = "api_token"
```

Comment:
Euh, maybe checkout a better name for api_token, in py-trello they are talking about api key, api secret, (oauth) token and (oauth) secret. so seeing api_token and api_key used both looks confusing

Reply:
Yeah, thinking about it now it might actually be an OAuth token. However, it seems weird that they would have you get an OAuth token manually, but maybe that isn't completely out of the ordinary.

I did go with the name "token", though, because that's what Trello themselves called it.

Reply:
yea but api_token api_key look similair, better make it normal token then

Reply:
"API Key" and "API Token" are both preexisting naming in [`strings.json`](https://github.com/home-assistant/core/blob/5a18de67e5593cb965f3f343a0dfbbcde6b427e5/homeassistant/strings.json#L46-L47). Also "token" by itself could have other meanings, so I'd like to be explicit with what it's a token for.